### PR TITLE
[RLlib] Release 1.5.1+: Patch for RLlib tf GPU broken.

### PR DESCRIPTION
**Pull request by sven1977**
Release 1.5.1: Patch for RLlib tf gpu broken.

Ray version 1.5.1 is currently broken for tf-GPU runs. This patch fixes the problem.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: src/ray/gcs/gcs_server/gcs_placement_group_manager.cc [hunk 1]
```diff
@@ -543,8 +543,9 @@ void GcsPlacementGroupManager::WaitPlacementGroup(
 }
 
 void GcsPlacementGroupManager::RetryCreatingPlacementGroup() {
-  execute_after(io_context_, [this] { SchedulePendingPlacementGroups(); },
-                RayConfig::instance().gcs_create_placement_group_retry_interval_ms());
+  execute_after(
```
Comment (sven1977) [inline src/ray/gcs/gcs_server/gcs_placement_group_manager.cc:546]: Just LINT related changes from here on.

**Comment by richardliaw** [comment]
I think this actually should land into 1.5.2, not 1.5.1. We shouldn't be updating released versions (in fact not sure if Pypi allows you to do that anyways)

cc @jiaodong @edoakes @zhe-thoughts are we doing a 1.5.2 release?

**Comment by jiaodong** [comment]
once this PR is merged i can help with releasing a 1.5.2 cut to address this issue.

**Review by richardliaw — CHANGES_REQUESTED** [review]
@sven1977 can we close this PR and make a new branch that targets releases/1.5.2 instead of 1.5.1?

**Pull request closed without merging**